### DISCUSSION TITLE
Looking for co-maintainers

### DISCUSSION THREAD

**Discussion by NotAShelf** [NotAShelf/nvf#132] [Announcements]
Hello all!

As you might have noticed, it has been a silent past few months for neovim-flake with most new features coming from outside contributors. I thank all of you who have contributed your changes, and those who have raised issue to point out issues that I have missed.

It's been a long difficult few months for me, with my work schedule only going worse - leading me to neglect the maintenance of neovim-flake, be it new feature or bugfixes. I wish to assure you that this flake is still on my mind, but I unfortunately do not have the time to maintain it as I would like to. 

As such I would like to announce that I am looking for co-maintainers that can help keep this flake up and running during my absence, which is set to last at least a few more months. Regardless of your experience with Nix, if you wish to help make a difference and willing to spend a little of your time on helping me maintain this project, I invite you to contact me over `raf@notashelf.dev` to become a co-maintainer. All I will ask from co-maintainers is to try and respond to active issues, review PRs and collaborate on me on existing PRs to help me progress a little faster by providing ideas where I cannot think of one.

**Comment by horriblename** [comment]
hey!

Thanks for creating and maintaining this project! I'd be happy to help with maintaining this project. If you need me for anything, you can DM me on discord `@hihl`.

**Comment by n3oney** [comment]
damn

**Reply by NotAShelf** [reply]
thanks neoney
